libexec/pyenv-global

```diff
@@ -1,16 +1,24 @@
 #!/usr/bin/env bash
 #
-# Summary: Set or show the global Python version
+# Summary: Set or show the global Python version(s)
 #
-# Usage: pyenv global <version>
+# Usage: pyenv global <version> <version2> <..>
 #
-# Sets the global Python version. You can override the global version at
+# Sets the global Python version(s). You can override the global version at
 # any time by setting a directory-specific version with `pyenv local'
 # or by setting the `PYENV_VERSION' environment variable.
 #
-# <version> should be a string matching a Python version known to pyenv.
-# The special version string `system' will use your default system Python.
-# Run `pyenv versions' for a list of available Python versions.
+# <version> should be a space-separated list of Python versions known
```

Comment:
In your example, `<version>` is not a space-separated list. It is already expanded as a list in the example.

Reply:
I agree, I think I was mincing over the right words too when I wrote that. It's just separate args on the CLI..